src/style.css

```diff
@@ -2,2 +1,92 @@
+:root {
+  --primary-color: #00acdc;
+}
+
+body,
+div,
+p,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+ul,
+ol,
+li,
+a,
+img,
+table,
+th,
+td {
   margin: 0;
+  padding: 0;
+  border: 0;
+  box-sizing: border-box;
+  font: inherit;
+  vertical-align: baseline;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.link_logo {
+  line-height: 0;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+}
+
+.nav_item {
+  position: relative;
+  margin-left: 20px;
+}
+
+.nav_item:first-child {
+  margin-left: 0;
+}
+
+.nav_link {
+  display: flex;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: black;
+  height: 60px;
+  align-items: center;
+}
+
+[data-qa='hover']:hover {
+  color: var(--primary-color);
+}
+
+.is-active {
+  color: var(--primary-color);
```

Comment:
The `.is-active` class changes the color of the active link. Ensure that this color contrast is sufficient for accessibility standards.